src/app/index.config.coffee

```diff
@@ -52,6 +52,18 @@
     )
   )
 
+  .config ($httpProvider) ->
+    $httpProvider.interceptors.push ($q, $injector, DEVISE_CONFIG) ->
+      {
+        request: (config) ->
+          # Intercept requests made to the API and reset the sessions timeout
+          if DEVISE_CONFIG.timeout_in > 0
+            if config.url.includes('mnoe') && !config.url.includes('sign_out') && !config.url.includes('sign_in')
```

Comment:
Is that more readable:
```coffeescript
if config.url.includes('mnoe') && !config.url.match(/(sign_out|sign_in)/)
```
Also shouldn't the timer be reset at `sign_in`?

Reply:
The timer does not need to be reseted on GET `/sign_in`. After a POST `sign_in`, the organization will be requested and reset the timer.
The regex isn't complex, I will use it